query-planner-js/src/composedSchema/removeInaccessibleElements.ts

```diff
@@ -110,23 +110,28 @@ export function removeInaccessibleElements(
     const newFieldMapConfig: GraphQLFieldConfigMap<any, any> =
       Object.create(null);
 
+    const errors = [];
     for (const [fieldName, fieldConfig] of Object.entries(fieldMapConfig)) {
       if (
         fieldConfig.astNode &&
         hasDirective(inaccessibleDirective!, fieldConfig.astNode)
       ) {
         continue;
       } else if (typesToRemove.has(getNamedType(fieldConfig.type))) {
-        throw new GraphQLError(
+        errors.push(new GraphQLError(
           `Field ${type.name}.${fieldName} returns ` +
             `an @inaccessible type without being marked @inaccessible itself.`,
           fieldConfig.astNode,
-        );
+        ));
       }
 
       newFieldMapConfig[fieldName] = fieldConfig;
     }
 
+    if (errors.length) {
```

Comment:
I know this works, but I still prefer to have the `> 0` on there for type checking.

Reply:
Oh totes! I even asked specifically about this on the fed2 PR and then totally forgot to carry the convention over here. Thanks for raising this one.

Reply:
Updated in https://github.com/apollographql/federation/pull/1581/commits/6b5e2feafceb4a908c2a477dfa39e1980465cbab